Multisig/App/SceneDelegate.swift

```diff
@@ -369,16 +369,18 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             return
         }
 
-        showWindow(makeEnterPasscodeWindow(showsCloseButton: true,
-                                           onPasscodeEnter: { [weak self] pwd in
-            try task(pwd)
-            self?.showMainContentWindow()
-        }, onError: { [weak self] error in
-            if let str = error as? String, str == "Cancelled" {
+        Task { @MainActor in
```

Comment:
What does this solve ?

Reply:
For some reason this was executed on a background thread. I just checked. Apparently I can remove this now, as it is no longer executed on a background thread.